sdk/cognitivelanguage/ai-language-text/test/public/analyzeBatch.spec.ts

```diff
@@ -1177,13 +1180,18 @@ matrix([["APIKey", "AAD"]] as const, async (authMethod: AuthMethod) => {
   describe(`[${authMethod}] TextAnalysisClient`, function (this: Suite) {
     let recorder: Recorder;
     let client: TextAnalysisClient;
+    let authoringClient: TextAuthoringClient;
 
     beforeEach(async function (this: Context) {
       recorder = await startRecorder(this.currentTest);
       client = createClient(authMethod, {
         resource: "CustomText",
         recorder,
       });
+      // Create client
+      const connectionString = process.env["ENDPOINT"] || "<connection string>";
```

Comment:
We shouldn't access env vars through `process` because it is a nodejs only module and it won't work in the browser. We should instead access them using the `env` definition exported by the recorder.